src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,223 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String carCreateQuery = "INSERT INTO `taxi_service`.`cars` (`model`, `manufacturer_id`) "
+                + "VALUES (?, ?);";
+        try (
+                Connection connection = ConnectionUtil
+                        .getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(carCreateQuery, Statement.RETURN_GENERATED_KEYS)
+                ) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create " + car + "." + e);
+        }
+        if (car.getDrivers() != null) {
+            setDriversByCar(car);
+        } else {
+            car.setDrivers(new ArrayList<>());
+        }
```

Comment:
If needed - do it inside `setDriversByCar`
```suggestion
            setDriversByCar(car);
```